test/unit/logic/account.js

```diff
@@ -376,36 +382,34 @@ describe('account', function () {
 			}, ['username'], function (err, res) {
 				expect(err).to.not.exist;
 				expect(res).to.have.length(50);
-				expect(_(res).map('username').dbSort()).to.eql(_.map(res, 'username'));
+				expect(res).to.eql(_.sortBy(res, 'username'));
 				done();
 			});
 		});
 
 		it('should ignore negative limit', function (done) {
-			var sortedUsernames = _(allAccounts).map('username').dbSort();
-
 			account.getAll({
 				limit: -50,
 				sort: 'username:asc'
 			}, ['username'], function (err, res) {
 				expect(err).to.not.exist;
-				expect(_(res).map('username').dbSort()).to.eql(_.map(res, 'username'));
+				expect(res).to.eql(_.sortBy(res, 'username'));
 				done();
 			});
 		});
 
 		it('should sort the result according to field type in ascending order', function (done) {
 			account.getAll({sort: 'username:asc'}, ['username'], function (err, res) {
 				expect(err).to.not.exist;
-				expect(_(res).map('username').dbSort()).to.eql(_.map(res, 'username'));
+				expect(res).to.eql(_.sortBy(res, 'username'));
 				done();
 			});
 		});
 
 		it('should sort the result according to field type in descending order', function (done) {
 			account.getAll({sort: 'username:desc'}, ['username'], function (err, res) {
 				expect(err).to.not.exist;
-				expect(_(res).map('username').dbSort('desc')).to.eql(_.map(res, 'username'));
+				expect(res).to.eql(_.sortBy(res, 'username').reverse());
```

Comment:
@karmacoma We recently change all occurrence of `sort` and `sortBy` and `orderBy` occurrences in all test cases to one method `dbSort`. So we should use same sorting pattern everywhere in test cases. I don' think reverting it back in one test file is a good idea.

Reply:
@nazarhussain I have reverted your PR to include only @diego-G changes.